specs/order_spec.rb

```diff
@@ -142,20 +143,29 @@
 
     it "Returns accurate information about the last order" do
       # TODO: Your test code here!
+      last = Order.find(100)
+      expect(last.id).must_equal 100
```

Comment:
So, what we are trying to do in these tests is make sure that the data that we have read in from the final order matches what was in our CSV. That way, we can trust that when we search for order 100, it matches the data that we had passed in in the first place. A more complete test might look like this:

```suggestion
      id = 100
      products = {
        "Amaranth" => 83.81,
        "Smoked Trout" => 70.6,
        "Cheddar" => 5.63,
      }
      customer_id = 20
      fulfillment_status = :pending

      order = Order.all.last

      expect(order.id).must_equal id
      expect(order.products).must_equal products
      expect(order.customer).must_be_kind_of Customer
      expect(order.customer.id).must_equal customer_id
      expect(order.fulfillment_status).must_equal fulfillment_status
```